### PR TITLE
Cu 3075b6m create a real footer with navigation 

### PR DESCRIPTION
**Pull request by matthew1809**
## Objective

What:
Why:
How:
Usage:

## Type

- [ ] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [ ] Style Tweaks
- [ ] Code Refactoring
- [ ] Documentation
- [ ] Tests

## Tested

Outline the approach you've taken to test the change

## Translation

- [ ] Have you translated all your Paragraphs, ButtonsTitle,... with i18n functionality?

## Documented

- [ ] Have you documented in changed file using comments
- [ ] Have you added instructions to the README if needed?

### CODE REVIEW

**Comment by diegolarraz** [comment]
Task linked: [CU-3075b6m Create a real footer with Navigation](https://app.clickup.com/t/3075b6m)

File: src/components/hits/components/CustomHits.jsx [hunk 1]
```diff
@@ -18,13 +18,13 @@ function CustomHits(props) {
...
   const [hits, setHits] = useState([]);
-  const hitsState = useRecoilValue(hitsAtom);
   const [hitsLoaded, setHitsLoaded] = useState(false);
   const productCard = useRef(null);
 
+
   // Decide whether to use hits from hook or props
   useEffect(() => {
-    // Check the props for the hits
+    // // Check the props for the hits
```
Comment (BenDanielGreep) [inline src/components/hits/components/CustomHits.jsx:27]: duplicate comment slashes

File: src/components/hits/components/injected-hits/InjectedHits.jsx [hunk 2]
```diff
@@ -34,14 +33,13 @@ const contentTypeComponentMap = {
 
 // This component renders the custom query hits, but also injects them with content from rule data or the injection Index
 const InjectedHits = (props) => {
-  // Get the regular hits
-  const { hits } = useInfiniteHits(props);
+  const { hits } = useInfiniteHits()
```
[the remark this answers is not in the document]
Comment (HugoWith) [reply src/components/hits/components/injected-hits/InjectedHits.jsx]: It has been outdated by merging with develop :)

File: src/components/searchresultpage/srpLaptop/SrpLaptop.jsx [hunk 3]
```diff
@@ -221,7 +226,13 @@ const SrpLaptop = () => {
                 <Configure hitsPerPage={1} page={0} />
               </Index>
               {/* Injected content*/}
-              <InjectedHits hitComponent={Hit} />
+              
+              <Index indexName={index}>
```
[the remark this answers is not in the document]
Comment (HugoWith) [reply src/components/searchresultpage/srpLaptop/SrpLaptop.jsx]: It has been outdated by merging with develop :)

**Pull request merged**